src/main/kotlin/me/gserv/fabrikommander/Common.kt

```diff
@@ -27,6 +27,9 @@ object Common : ModInitializer {
         HomesCommand(dispatcher).register()
         SetHomeCommand(dispatcher).register()
 
+        // Teleport commands
+        TpaCommand(dispatcher).register()
```

Comment:
Each command should have its own class, don't stack commands in the same class.

Reply:
Come to think of it, the class should also be along the lines of `TeleportRequestCommand` as well.